lib/publiccloud/instance.pm

```diff
@@ -372,7 +372,6 @@ Parameters:
                          (e.g. when cloud-init state is cleared)
  username => default: username().
  systemup_check => If true, checks if the system is up too, instead of just checking the ssh port; default: !wait_stop.
- scan_ssh_host_key => If true, rescan the SSH host key (that's alsos needed when host changed its address)
```

Comment:
why you removed this line ? I still see argument in the code

Reply:
See line `368` - this was a duplicate.